src/main/java/rx/subscriptions/BooleanSubscription.java

```diff
@@ -28,9 +28,7 @@
 public final class BooleanSubscription implements Subscription {
 
     private final Action0 action;
```

Comment:
Changed in a separate commit here: https://github.com/ReactiveX/RxJava/commit/0bdba460f7dd1ec1e00fc881645bb5c071a23705. Is that what you had in mind?